src/pybind/mgr/dashboard_v2/module.py

```diff
@@ -143,12 +143,31 @@ class ApiRoot(object):
         def __init__(self, mgrmod):
             self.ctrls = load_controllers(mgrmod)
             logger.debug('Loaded controllers: {}'.format(self.ctrls))
-            for ctrl in self.ctrls:
+
+            first_level_ctrls = [ctrl for ctrl in self.ctrls
+                                 if '/' not in ctrl._cp_path_]
+            multi_level_ctrls = set(self.ctrls).difference(first_level_ctrls)
+
+            for ctrl in first_level_ctrls:
                 logger.info('Adding controller: {} -> /api/{}'
                             .format(ctrl.__name__, ctrl._cp_path_))
                 ins = ctrl()
```

Comment:
Can you please rename this variable to ``inst`` or ``instance`` (i know it's not part of this PR). The current name doesn't enlight me what it is for on the first look.